Exercícios/Para casa/projeto/BankAccount.js

```diff
@@ -101,7 +101,13 @@ class BankAccount {
 
 	// Criar método cashWithdrawal
 	cashWithdrawal(amount) {
-		// Implementar esse método
+        if (this.#balance >= amount) {
+			
+            this.#balance -= amount;
+			console.log(`O saldo atual da conta é de R$ ${this.#balance}`);
```

Comment:
Eu fiz o meu método bem parecido com o seu, porém eu chamei dentro o outro método da Classe BankAccount - debitAmount(amount).
Para chamá-lo de dentro de outra função usei o this.debitAmount(amount). 

;)

Poderia ser uma alternativa para esse trechinho do código não ficar repetido, já que esse método já faz os cálculos de um saque!